packages/@o3r/components/tsconfig.builders.json

```diff
@@ -16,8 +16,7 @@
   ],
   "exclude": [
     "**/*.spec.ts",
-    "**/templates/**/*.ts",
-    "**/mocks/**/*.ts",
-    "schematics/**/templates/**"
+    "**/templates/**",
+    "**/mocks/**/*.ts"
```

Comment:
Why do we keep the `*.ts` for `mocks` but not for `templates`

Reply:
Not needed to keep it but I just fixed the `/template/` pattern without touching others